copter/source/docs/systemid-mode-operation.rst

```diff
@@ -95,3 +95,101 @@ main loop frequency divided by a sub-sample factor.
+This enables the analysis of the closed-loop behaviour of the respective axis as well as the identification of the axis dynamic behaviour by using the PID controller output as excitation signal of the system.
+The next section discusses the results of the presented approach.
+
+Results for the Pitch Axis
+-----------------------------------------
+
+The approach described above has been performed for the pitch axis of a multicopter and the results are presented bellow.
+The system identification flight was performed with the following settings:
+
++--------------------------------------+------------------+
+| Parameter                            | Value            |
++--------------------------------------+------------------+
+| :ref:`SID_AXIS<SID_AXIS>`            | 8                |
++--------------------------------------+------------------+
+|:ref:`SID_MAGNITUDE<SID_MAGNITUDE>`   | 130 (deg/s)      |
++--------------------------------------+------------------+
+|:ref:`SID_F_START_HZ<SID_F_START_HZ>` | 0.05 Hz          |
++--------------------------------------+------------------+
+|:ref:`SID_F_START_HZ<SID_F_STOP_HZ>`  | 5 Hz             |
++--------------------------------------+------------------+
+|:ref:`SID_T_FADE_OUT<SID_T_FADE_OUT>` | 0 s              |
++--------------------------------------+------------------+
+|:ref:`SID_T_FADE_IN<SID_T_FADE_IN>`   | 5 s              |
++--------------------------------------+------------------+
+| :ref:`SID_T_REC<SID_T_REC>`          | 70 s             |
++--------------------------------------+------------------+
+
+The next :ref:`figure<fig-io-signals-axis8>` shows the recorded input (pitch rate target) and output (measured pitch rate) signals over time.
+Below that, the power spectral densities of both signals are depicted, indicating how the power of the signal is distributed over the tested frequency range.
+Examining the first 45 seconds of the test flight, it can clearly be observed that there is not much of a response on the pitch axis, as its rate stays almost constant around 0 rad/s.
+Note that the initial frequency of the sweep (0.05 Hz) is hold for 40 seconds at the beginning of the test flight in order to run two full cycles with a low frequency.
+With the increase of the frequency afterwards, the magnitude of the resulting pitch rate also rises.
+
+.. image:: ../images/PitchRateAxis8IdentificationIOSignals.png
+:name: fig-io-signals-axis8
+
+By estimating the :ref:`frequency-response<fig-freq-resp>` of the system and plotting the Bode diagram, the described behaviour above becomes clear.
+For lower frequencies, the system response reacts with low amplitudes (magnitude decreases).
+Normally, one would expect a (nearly) constant gain around 1 (0 dB) of the closed-loop transfer function for low frequency excitations and a decreasing magnitude for high frequences (which does not show in the diagram because the highest frequency of the frequency-sweep was 5 Hz).
```

Comment:
@amilcarlucas @fbredeme  The bode plot makes perfect sense now that I read your approach to obtaining the data.  In stabilize mode the user is requesting a desired attitude and the flight controller will try to maintain that desired attitude.  When you inject pitch, roll or yaw rates, the flight controller sees theses as disturbances and quickly tries to attenuate them through the attitude controller feedback.  Thus your bode plot shows reduced gain at low frequencies until you get above the response frequency of the attitude controller which is based on the Angle P gain.

Reply:
What we are looking at is the frequency response of the closed-loop of the rate controller though. The injected sweep is added to the desired rate and thus to the desired value of the controller. So, the injected sweep does not pose a disturbance for the rate controller but only for the attitude controller. From my understanding, we would see the attentuation if we looked at the frequency response of the attitude controller, but not for the rate controller, since the rate controller is not exposed to an injected disturbance and just tries to follow the desired rate

Reply:
Even though you are making inputs into desired rate for the rate controller, the attitude controller is still maintaining the desired attitude based on stick input. The attitude controller will see the changes in the response due to the change in desired rate and will then make a counter input to oppose that rate in order to make the attitude remain at what was commanded by the pilot. 

Rate and attitude are kinematically linked so if you see an attenuation in rate then you would see the same in attitude

Use the SITL and perform a frequency sweep as I said above and then perform the spectral analysis on the rate controller. You will see the low frequency will not be attenuated. Or you can do it with rare inputs and just make the Angle P gain zero.